xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -244,8 +244,10 @@ bool CPeripheralCecAdapter::InitialiseFeature(const PeripheralFeature feature)
     if (m_configuration.serverVersion < CEC_LIB_SUPPORTED_VERSION)
     {
       /* unsupported libcec version */
-      CLog::Log(LOGERROR, g_localizeStrings.Get(36040),
-                m_cecAdapter ? m_configuration.serverVersion : -1, CEC_LIB_SUPPORTED_VERSION);
+      CLog::Log(
+          LOGERROR,
+          "Detected version of libCEC interface ({0:x}) is lower than the supported version {1:x}",
```

Comment:
This seems like a step backwards in localisation. Without Log handling the string case, i think youd be better to go back to c_str() rather than hardcode the string like this.

Reply:
I think this is the only log line that uses localization that I can see

Reply:
We never localize log messages.